catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventHandlerDelegate.java

```diff
@@ -167,7 +166,8 @@ public Metacard getMetacard(String id) {
          */
         if (metacard.getAttribute(attribute.getName()) == null) {
           metacard.setAttribute(attribute);
-        } else if (MapUtils.getBoolean(multiValuedMap, attribute.getName(), false)) {
+        } else if (multiValuedMap.getOrDefault(attribute.getName(), false)) {
+          metacard.getAttribute(attribute.getName()).getValues().removeAll(attribute.getValues());
```

Comment:
❓ Why do we do a remove all immediately followed by an add all?

Reply:
Quick and dirty union of the values so we don't get duplicates.